DEVELOP.md

```diff
@@ -0,0 +1,485 @@
+    ├── daffodil-cli/
+    │   ├── README.md          - Explains how to start Daffodil
+    │   ├── bin.LICENSE        - Contains licenses of Daffodil and subcomponents
+    │   ├── bin.NOTICE         - Contains notices of Daffodil and subcomponents
+    │   ├── build.sbt          - Tells sbt how to build CLI and installers
+    │   └── src/               - Contains CLI source code, tests, scripts, etc.
+    ├── daffodil-core/
+    │   └── src/               - Contains Daffodil's core
+    ├── daffodil-io/
+    │   └── src/               - Contains Daffodil's input/output/charsets
+    ├── daffodil-japi/
+    │   ├── build.sbt          - Tells sbt how to generate javadocs
+    │   └── src/               - Contains Daffodil's Java API
+    ├── daffodil-lib/
+    │   └── src/               - Contains Daffodil's utilities
+    ├── daffodil-macro-lib/
+    │   └── src/               - Defines Daffodil's Scala 2.x macros
+    ├── daffodil-propgen/
+    │   └── src/               - Contains generators to generate more source code
+    ├── daffodil-runtime1/
+    │   └── src/               - Contains Daffodil's Scala parser
+    ├── daffodil-runtime1-layers/
+    │   └── src/               - Contains Daffodil's Scala layer transformers
+    ├── daffodil-runtime1-unparser/
+    │   └── src/               - Contains Daffodil's Scala unparser
+    ├── daffodil-runtime2/
+    │   └── src/               - Contains Daffodil's C code generator
+    ├── daffodil-sapi/
+    │   └── src/               - Contains Daffodil's Scala API
+    ├── daffodil-schematron/
+    │   ├── README.md          - Explains how to validate with Schematron rules
+    │   └── src/               - Contains Daffodil's Schematron validator
+    ├── daffodil-tdml-lib/
+    │   └── src/               - Contains Daffodil's TDML runner
+    ├── daffodil-tdml-processor/
+    │   └── src/               - Contains Daffodil's Scala & C TDML implementations
+    ├── daffodil-test/
+    │   └── src/               - Contains Daffodil's TDML tests
+    ├── daffodil-test-ibm1/
+    │   └── src/               - Contains more TDML tests from IBM & Tresys
+    ├── daffodil-udf/
+    │   ├── README.md          - Explains how to implement user defined functions
+    │   └── src/               - Contains Daffodil's UDF interfaces 
+    ├── project/
+    │   ├── Dependencies.scala - Defines third party dependencies
+    │   ├── Rat.scala          - Lists files to be ignored by license audit tool
+    │   ├── build.properties   - Defines which sbt version to use
+    │   └── plugins.sbt        - Adds third party plugins to sbt
+    ├── test-stdLayout/
+    │   └── src/               - Contains TDML tests laid out in different ways
+    └── tutorials/
+        └── src/               - Contains Daffodil's TDML test tutorials
+
+When compiled into a command line interface, Daffodil is composed of a
+script, a log4j2 configuration file, 14 Daffodil jars, and 37
```

Comment:
I would avoid using specific numbers, they are going to quickly go out of date and I'm not sure really provides additional value.